.github/workflows/crib-integration-test.yml

```diff
@@ -40,24 +36,6 @@ jobs:
           gc-host: ${{ secrets.GRAFANA_INTERNAL_HOST }}
           gc-org-id: ${{ secrets.GRAFANA_INTERNAL_TENANT_ID }}
 
-      - name: setup-gap k8s
```

Comment:
I believe this is necessary but @skudasov can confirm.

Reply:
I wasn't able to find in the code, but yeah he may be pulling some statuses from the API.

Reply:
It is necessary, how we then connect? We connect using GAP headers through sigv4proxy @njegosrailic

Reply:
I connect here
```
          GAP_URL: ${{ secrets.GAP_URL }}
```

Reply:
Thanks.